Confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -66,9 +66,7 @@ class Confidence internal constructor(
 
     private suspend fun resolve(flags: List<String>): Result<FlagResolution> {
         debugLogger?.let {
-            for (flag in flags) {
-                debugLogger.logFlags("ResolveFlag", flag)
-            }
+            debugLogger.logFlag("Resolve")
```

Comment:
Do we want to explain why first resolve will be empty? Or is this more of a future fix/refactor territory?

Reply:
I'm thinking we can provide a new PR where we clean up the API (`resolve` functions should not take the list of flags).